photospp.sh

```
@@ -15,9 +15,7 @@ prefer_system_check: |
   ls $PHOTOSPP_ROOT/ > /dev/null && \
   ls $PHOTOSPP_ROOT/include/Photos > /dev/null && \
   ls $PHOTOSPP_ROOT/lib > /dev/null && \
-  ls $PHOTOSPP_ROOT/lib/libPhotospp.so > /dev/null && \
-  ls $PHOTOSPP_ROOT/lib/libPhotosppHEPEVT.so > /dev/null && \
-  ls $PHOTOSPP_ROOT/lib/libPhotosppHepMC3.so > /dev/null
+  ls $PHOTOSPP_ROOT/lib/libPhotospp.so > /dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)